sources/packages/api/src/database/entities/education-program.model.ts

```diff
@@ -148,7 +148,7 @@ export class EducationProgram extends RecordDataModel {
   @Column({
     name: "has_minimun_age",
   })
-  hasMinimunAge?: string;
+  hasMinimunAge?: boolean;
```

Comment:
Can we take advantage of these changes and fix the type on _hasMinimunAge_?

Reply:
Fixed.